pipeline-runner/tests/testthat/test-gem2s-6-prepare_experiment.R

```diff
@@ -88,11 +106,71 @@ test_that("prepare_experiment adds 0 indexed cell_ids and other metadata to scda
   expect_equal(added_ids, expected_ids)
 })
 
-
 test_that("prepare_experiment generates qc_config that matches snapshot", {
   prev_out <- mock_prev_out()
   input <- list(experimentId = "1234")
   task_out <- prepare_experiment(input, NULL, prev_out)$output
 
   expect_snapshot(str(task_out$qc_config))
 })
+
+test_object <- function() {
```

Comment:
This is because the step of creating the seurat object is fairly slow, and I wanted to do it just once. It's gonna slow down the tests even more if we separate this.